### ISSUE TITLE
Is there a way to make oneko not follow the mouse

### ISSUE THREAD

**Issue by JadeVane** [IreneKnapp/oneko#2]
Provide an option for oneko to play in one place by itself, so it can walk, tick, or sleep anywhere on the screen by itself, instead of running with the mouse all the time.

**Comment by IreneKnapp** [comment]
That is a really neat idea! I love it! It would require some creativity to come up with, like, how does the kitten decide what to do, how long does it stay doing it, that sort of thing. I'm going to chew on this. Let me know if you have more thoughts on how it would work. :)

**Comment by JadeVane** [comment]
Thank you for your quick reply, I am willing to share my idea what i can do if it can help. so there are some ideas that maybe provide some reference

1. **Default mode: keep running or ticking**
Let neko keep running or ticking somewhere when I move the mouse frequently, which means that the interval between two mouse movements is less than 5 seconds.
1. **Quiet mode: sleeping or ticking**
I need to foucs on my work when I am reading, typing or watching a video, so it's better to let oneko sleeping or ticking in one place, and do not run to anywhere.
1. **Play mode: something interesting**
I want to play with oneko when I want to take a rest or just feel boring, so it may be interesting to let it be dizzy by my rotating mouse, be clicked continuously to sneeze, or get caught by the mouse somewhere, then it will run after the mouse to catch it and prevent me from using it, for example, it has to be hung on the mouse all the time that I need to shake the mouse to get rid of it, or take the mouse away, If possible, this state will last for 2 to 3 seconds. 

I am not sure if a oneko can do this, just feel free to tell anything in your mind.

Actually the latest version of oneko do a great job, and I have been searching for something like this for servel years, thank you for making such a great project.

**Comment by IreneKnapp** [comment]
I'm proud to be the maintainer of oneko, but I didn't make it. It dates to the 90s, and if you look at the change history, basically each new version has been by a different author. It had been unchanged for nearly twenty years when I took over as maintainer. So I like to think of it as my contribution to something that's larger than me.

These are good thoughts, thank you. :)

**Comment by JadeVane** [comment]
Indeed, I don’t speak Japanese, so I don’t know much about the content of the introduction. If possible, it might be better to provide an English version of the readme.

**Comment by IreneKnapp** [comment]
Yes, I'm certain you're right. I guess I can add that to my todo list. :)
